tests/attention/test_cute_dsl_fmha_jit.py

```
@@ -0,0 +1,441 @@
+        torch.bfloat16,
+        5e-2,
+    ),
+}
+
+
+@pytest.mark.parametrize("mode", list(_NONBS_MODES))
+@pytest.mark.parametrize("causal", [False, True])
+def test_cute_dsl_fmha_jit(mode, causal):
+    """Non-block-scaled varlen prefill, JIT-compiled from the vendored kernel."""
+    torch.manual_seed(0)
+    qk_dt, v_dt, out_dt, out_torch_dt, atol = _NONBS_MODES[mode]
```

Ruff (0.15.20)

[warning] Unpacked variable `out_torch_dt` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)